kratos/geometries/geometry.h

```diff
@@ -1337,6 +1337,44 @@ class Geometry : public PointerVector<TPointType>
 
         return rResult;
     }
+    
```

Comment:
@RiccardoRossi Done, it was the same way in the previous one too, I corrected both